cdc/owner/ddl_manager.go

```diff
@@ -332,12 +333,14 @@ func (m *ddlManager) executeDDL(ctx context.Context) error {
 		return nil
 	}
 
-	// If changefeed is in BDRMode, skip ddl.
-	if m.BDRMode {
-		log.Info("changefeed is in BDRMode, skip a ddl event",
+	// If changefeed is in BDRMode and the ddl is not sent by BDRRolePrimary role, skip it.
```

Comment:
Please explain that TiCDC will capture ddls from secondary and local_only cluster, and ignore such events here.